sdk/keyvault/keyvault-admin/src/accessControlClient.ts

```diff
@@ -2,16 +2,10 @@
 // Licensed under the MIT license.
 /// <reference lib="esnext.asynciterable" />
 
-import {
-  TokenCredential,
-  isTokenCredential,
-  signingPolicy,
-  createPipelineFromOptions,
-  InternalPipelineOptions
-} from "@azure/core-http";
+import { TokenCredential } from "@azure/core-auth";
 import { PagedAsyncIterableIterator } from "@azure/core-paging";
 
-import { challengeBasedAuthenticationPolicy, createTraceFunction } from "../../keyvault-common/src";
+import { createTraceFunction } from "./tracingHelpers";
```

Comment:
Yup that is the goal - since we're only moving one there will be some temporary duplication but once we move them all we will move this back to KV common